src/main/java/cc/woverflow/hytils/handlers/language/LanguageData.java

```diff
@@ -45,7 +45,7 @@ public class LanguageData {
     private String chatCleanerMysteryBoxFind = ".+(?<player>\\w{1,16}) found (?:a|an) (?:✰{5} Mystery Box!|.+ in a .+ Mystery Box!)$";
     private String chatCleanerSoulWellFind = "^.+ has found .+ in the Soul Well!$";
     private String chatCleanerGameAnnouncement = "^\u27A4 A .+ game is (?:available to join|starting in .+ seconds)! CLICK HERE to join!$";
-    private String chatCleanerBedwarsPartyAdvertisement = "(?<number>[1-3]/[2-4])";
+    private String chatCleanerBedwarsPartyAdvertisement = "(?<number>[1-7]/[2-8])";
```

Comment:
can you update this in our data repo (https://github.com/W-OVERFLOW/DataStorage) please ty